docker-compose fails building mqueryfront

**Environment information**
* Mquery version (from the /status page):
v1.2 (git clone)
* Ursadb version (from the /status page):
v1.4.2
* Installation method:
  - [x] Generic docker-compose
  - [ ] Dev docker-compose
  - [ ] Native (from source)
  - [ ] Other (please explain)

**Reproduction Steps**

Basically followed the docker-compose up commands:

```
$ git clone --recurse-submodules https://github.com/CERT-Polska/mquery.git
$ cd mquery
$ vim .env  # optional - change samples and index directory locations
$ docker-compose up --scale daemon=3  # building the images will take a while
```
**Expected behaviour**
Running docker-compose setup.

**Actual behaviour the bug**

Failed to build the web docker image due to npm building issues.

**Screenshots**

The build step for the failing `npm install && npm run build`:
```
Step 5/14 : RUN npm install && npm run build
 ---> Running in 75971a7a6c0a

> @fortawesome/fontawesome-common-types@0.2.34 postinstall /app/node_modules/@fortawesome/fontawesome-common-types
> node attribution.js

Font Awesome Free 0.2.34 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)


> uglifyjs-webpack-plugin@0.4.6 postinstall /app/node_modules/uglifyjs-webpack-plugin
> node lib/post_install.js


> @fortawesome/fontawesome-svg-core@1.2.34 postinstall /app/node_modules/@fortawesome/fontawesome-svg-core
> node attribution.js

Font Awesome Free 1.2.34 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)


> @fortawesome/free-solid-svg-icons@5.15.2 postinstall /app/node_modules/@fortawesome/free-solid-svg-icons
> node attribution.js

Font Awesome Free 5.15.2 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)

npm WARN @monaco-editor/react@3.8.3 requires a peer of monaco-editor@^0.21.2 but none is installed. You must install peer dependencies yourself.
npm WARN @monaco-editor/loader@0.1.3 requires a peer of monaco-editor@^0.21.2 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 1469 packages from 796 contributors and audited 1538 packages in 32.875s

4 packages are looking for funding
  run `npm fund` for details

found 1368 vulnerabilities (680 low, 14 moderate, 673 high, 1 critical)
  run `npm audit fix` to fix them, or `npm audit` for details

> mqueryfront@0.1.0 build /app
> react-scripts build

Creating an optimized production build...
Failed to compile.

./node_modules/@monaco-editor/react/lib/es/ControlledEditor/ControlledEditor.js
Module parse failed: Unexpected token (11:2)
You may need an appropriate loader to handle this file type.
|   onChange,
|   editorDidMount,
|   ...props
| }) {
|   const editor = useRef(null);


npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! mqueryfront@0.1.0 build: `react-scripts build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the mqueryfront@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-02-23T15_32_12_117Z-debug.log
The command '/bin/sh -c npm install && npm run build' returned a non-zero code: 1

```

**Additional context**

```

```

--- a/src/mqueryfront/package.json
+++ b/src/mqueryfront/package.json
@@ -4,7 +4,7 @@
     "private": true,
     "dependencies": {
         "@babel/runtime": "^7.9.2",
-        "@monaco-editor/react": "^3.3.0",
+        "@monaco-editor/react": "3.7.5",
         "axios": "^0.18.0",
         "bootstrap": "^4.2.1",
         "filesize": "^4.0.0",

